### ISSUE TITLE
Instructions for installing `sdc-docker-setup.sh` locally

### ISSUE THREAD

**Issue by foxx** [joyent/sdc-docker#90]
There are no instructions in the README for installing `sdc-docker-setup.sh` locally. There's a blog post [here][1] with some better instructions, perhaps we should copy this over to README?

[1]: https://www.joyent.com/blog/container-service-preview

**Comment by misterbisson** [comment]
@foxx you're right to point out the inadequacies of the docs here. For now, the following might help:

1. SmartDataCenter (SDC) was renamed Triton (as noted in https://github.com/joyent/triton#triton-datacenter). Please look at the docs for Triton for any questions about using Triton Enterprise or open source, https://www.joyent.com/datacenter/docs/enterprise.
2. The Triton CLI is the preferred method to configure Docker clients for all cases. See https://docs.joyent.com/private-cloud/install/client-setup for detailed docs on how to do so.
3. `sdc-docker-setup.sh` exists now only as a simplified code example of what needs to be done to configure Docker clients.

**Comment by foxx** [comment]
Thanks for the clarification, this confirms what I had already discovered. I think there will certainly need to be a docs clean up in future, there's a lot of conflicting and out of date information stemming from organic growth of the platform, and I suspect this is just one instance of many that could be improved. Should I continue to raise individual docs errors like this, or can I assume that they are being mass cleaned at some point in the future?

**Comment by misterbisson** [comment]
> Should I continue to raise individual docs errors like this, or can I assume that they are being mass cleaned at some point in the future?

Continue raising, please.

**Comment by bahamat** [comment]
@misterbisson there is still one lingering edge case where the user has encrypted SSH keys.
